corehq/apps/translations/static/translations/js/translations.js

```diff
@@ -240,15 +234,11 @@ var mk_translation_ui = function (spec) {
                 translation = translation_ui.translations[key];
                 translation.ui.appendTo($list_tbody);
             }
-        } else if (!translation_ui.edit) {
```

Comment:
This I think this needs to be `else` because we still want to display no translations if there aren't any